tools/not.cpp

```diff
@@ -0,0 +1,86 @@
+//===- not.cpp - The 'not' testing tool -----------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// Usage:
+//   not cmd
+//     Will return true if cmd doesn't crash and returns false.
+//   not --crash cmd
+//     Will return true if cmd crashes (e.g. for testing crash reporting).
+
+// This file is a stripped down version of not.cpp from llvm/utils. This does
+// not depend on any LLVM library.
+
+#include <cstdlib>
+#include <iostream>
+#include <sstream>
+
+#ifdef _WIN32
+#include <windows.h>
```

Comment:
nits: add ... before `#include <windows.h> to reduce the size of the Win32 header files included.
```cpp
#define WIN32_LEAN_AND_MEAN
#define NOMINMAX
#include <windows.h>
```

Reply:
Done